cypress/fixtures/article_category_list.json

```diff
@@ -0,0 +1,88 @@
+{
+  "articles": [
+    {
+      "id": 1,
+      "title": "title 1",
+      "category": "sport",
+      "published_at": "2020-05-24 14:14",
+      "image": "https://picsum.photos/400/300"
+    },
```

Comment:
This fixture file is not up-to-date.

Reply:
I added the image, API readme did not have it